docs/writing-easyconfig-files.md

```diff
@@ -940,6 +940,70 @@ Example:
     eb WRF-3.5.1-ictce-5.3.0-dmpar.eb --try-toolchain=intel,2014b -r
     ```
 
+## Updating existing easyconfigs for another toolchain
+
+While `try-toolchain` is a convenient way of testing existing software versions with another toolchain
+you rarely want this behavior.
+Usually when using a newer toolchain you also want to use newer versions of (at least) the dependencies.
+Often there are already at least easyconfigs available for the dependencies in the new toolchain,
+and especially if those are already installed you should use those as dependencies for the updated easyconfig.
+
+So the process of updating an easyconfig to a newer toolchain version could look like:
+
+1. Copy the easyconfig and change the toolchain version
+1. Usually, especially when updating to the latest toolchain,
+   you should search for the most recent version of the software too and use that.
+1. For each listed dependency find the easyconfig for the new toolchain version.
+   Here you need to take [toolchain hierarchy](common-toolchains.md#toolchains_diagram) into account.
+   I.e. a dependency of an easyconfig for the `foss` toolchain might use the `GCC` or `GCCcore` toolchain.
+   Check the toolchain easyconfig file (e.g. `foss-2023b.eb`) to find the versions of the sub-toolchain.
+   For very recent toolchains you can also check the `develop` branch of the [easyconfigs git repository](https://github.com/easybuilders/easybuild-easyconfigs) and open PRs.
+1. If an easyconfig of the dependency for the new toolchain already exists use its version in your new easyconfig.
+   Otherwise, you need to create a new easyconfig for the new toolchain version based on that of another toolchain version.
+   The steps are the same and need to also be repeated for each dependency of this dependency.
```

Comment:
```suggestion
1. If an easyconfig for the listed dependency with the new toolchain already exists use its version in your new easyconfig.
   Otherwise, you need to create a new easyconfig for the listed dependency with the new toolchain version (based on that of another toolchain version).
   The steps are the same and also need to be repeated for each dependency of this dependency.
```